### PR TITLE
[node][next] Bump node-file-trace to 0.8.0

### PR DESCRIPTION
**Pull request by styfle**
Bump `node-file-trace` to version [0.7.0](https://github.com/vercel/node-file-trace/releases/tag/0.7.0) and then [0.8.0](https://github.com/vercel/node-file-trace/releases/tag/0.8.0)

- Add support for `exports`, dynamic import from CJS support, package boundary emission
- Fix trailing slash in `require()`
- Import named instead of default

### CODE REVIEW

**Review by Timer — DISMISSED** [review]

**Review by ijjk — DISMISSED** [review]

**Review by TooTallNate — DISMISSED** [review]

**Review by Timer — DISMISSED** [review]

**Review by ijjk — DISMISSED** [review]

**Comment by styfle** [comment]
@ijjk @Timer  It looks like `next@canary` just broke `14-next-offline`.

image

I can reproduce locally by running `next build` so the failure is not related to nft.

**Comment by styfle** [comment]
I bumped next to latest (see cb038581c7ce12f36f8632456eb57ca264217162) so that the canary bug can be fixed in a separate PR and we can unblock this PR.

**Comment by timneutkens** [comment]
@styfle I've fixed the issue here: https://github.com/vercel/next.js/pull/15311 👍

**Review by timneutkens — DISMISSED** [review]

File: packages/now-next/test/fixtures/14-next-offline/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "dependencies": {
-    "next": "canary",
+    "next": "latest",
```
Comment (Timer) [inline packages/now-next/test/fixtures/14-next-offline/package.json]: Can we revert this change? We fixed this test with `next@canary`!

Comment (styfle) [reply packages/now-next/test/fixtures/14-next-offline/package.json]: Done!

**Review by Timer — APPROVED** [review]

**Review by lucleray — APPROVED** [review]

**Review by timneutkens — APPROVED** [review]
